apps/work/types.py

```diff
@@ -0,0 +1,74 @@
+import strawberry
+from typing import List, Optional
+from strawberry import auto
+from strawberry.types import Info
+
+from utils import build_url
+from .models import Work, WorkImage, People
+from gaatha.types import FileFieldType
+from .filters import WorkFilter
+from gaatha.enums import GenericEnumValue
+
+
+@strawberry.django.type(WorkImage)
+class WorkImageType:
+    id: auto
+
+    @strawberry.field
+    async def image(self, info: Info) -> Optional[FileFieldType]:
```

Comment:
For new version of python, this should work
```suggestion
    async def image(self, info: Info) -> FileFieldType | None:
```